src/components/binding/binding-panel.js

```diff
@@ -213,13 +187,44 @@ genotet.BindingPanel.prototype.updateTracks = function() {
       this.container.find('#genes #track-' + index).remove();
     }
   }
+
+  this.signal('loadBindingList');
+
+  this.container.find('#genes .glyphicon-remove')
+    .css('display', this.data.tracks.length == 1 ? 'none' : '');
+};
+
+/**
+ * Updates the track list for panel.
+ */
+genotet.BindingPanel.prototype.updateTrackList = function() {
```

Comment:
I know. I should combine the DOM manipulations in the same function. However, you **ignored** the load binding list part, and **just suggested to combine when you see** the DOM manipulations are separated. You had better suggest to "combine DOM manipulations" in this case, not just "combine these two functions". Otherwise, I would feel very bad.